fabric-rendering-fluids-v1/src/main/java/net/fabricmc/fabric/api/client/render/fluid/v1/FluidTextureRegistry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.render.fluid.v1;
+
+import net.minecraft.screen.PlayerScreenHandler;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.event.client.ClientSpriteRegistryCallback;
+
+/**
+ * Implements the registration for fluid textures.
+ */
+public interface FluidTextureRegistry {
+	/**
+	 * Registers the fluid textures.
+	 *
+	 * @param textures Textures to register.
+	 */
+	static void register(Identifier... textures) {
```

Comment:
IMO people can just use the sprite callback directly without this wrapper method

Reply:
The scope is to avoid the copy-paste of the callback method.
This way I think makes more comprehensible that you are actually registering a texture and is aligned with all registration methods for blocks, items...

Reply:
I don't really see what there is to copy-paste? The difference between these two is really small IMO:

```java
// fabric-textures-v0 directly
ClientSpriteRegistryCallback.event(PlayerScreenHandler.BLOCK_ATLAS_TEXTURE)
    .register((atlasTexture, registry) -> registry.register(MY_TEXTURE));

// FluidTextureRegistry
FluidTextureRegistry.register(MY_TEXTURE);
```

It also matches how you'd register textures for any other blocks that go around JSON model loading (that don't just bind that texture directly in a BER). Anyway, others' opinions are welcome as well of course :smile:

(Also, this is not really "aligned with" any registration method for blocks or items - there you define the textures in model files written in json or using this exact callback)

Reply:
It's only a simplification to make the registration similar to Registry.register(Registry.BLOCK...) it's obvious that is more 
immediate to learn and clean (i mean the code), that to register a texture you must call a FluidTextureRegistry.register(...), instead to register an event in ClientSpriteRegistryCallback and implement an anonymous interface...

However let's wait other opinions, of course, to maintain or remove this...
Actually it is my obsession to always clean up things 😅

Reply:
Ok after thinking about it, considering that no one has ever complained about this, and that is not the job for rendering-fluids, but textures module, I've decided to remove this.

Actually is only a simplification that I've added, trying to cleanup the code... it is my obsession to always try to clean up! 😅